projects/news_feed/psl_matches.py

```diff
@@ -59,8 +59,11 @@ def main():
     psl_json_data = simplejson.dumps(all_fetched_information)
 
     # creating a separate json file named as psl_json_data
-    new_file = open('psl_json_data.txt', 'w')
-    new_file.write(psl_json_data)
+    psl_json_data_file = open('E:/TechCrunch/projects/news_feed/data/psl_json_data.txt', 'w')
+    psl_json_data_file.write(psl_json_data)
+
+    # printing that the psl data has been saved in psl_json_data file
+    print("PSL matches data saved in psl_json_data file")
```

Comment:
use `psl_json_data_file` varaible in print 

`print("PSL matches data saved in {} file".format(psl_json_data_file))`

Reply:
done